src/app/static/src/tests/components/runWorkflow/runWorkflow.test.ts

```diff
@@ -92,11 +92,18 @@ describe(`runWorkflow`, () => {
         await wrapper.find("#rerun").trigger("click")
         expect(wrapper.find(workflowWizard).exists()).toBe(true)
         expect(wrapper.vm.$data.workflowStarted).toBe(true)
+        expect(wrapper.find(workflowWizard).props("isRerun")).toBe(true)
 
         expect(wrapper.find("#run-header").text()).toBe("Run workflow")
         const buttons = wrapper.findAll("button")
         expect(buttons.at(0).text()).toBe("Cancel")
         expect(buttons.at(1).text()).toBe("Run workflow")
+        expect(wrapper.find("#workflow-name-div input").attributes("readonly")).toBe("readonly")
+
+        expect(wrapper.find("#next-workflow").attributes("disabled")).toBe("disabled")
```

Comment:
Shouldn't the button be enabled immediately for a re-run? User can't change name, changelog isn't mandatory and initial database instances should be pre-selected to first in list. Need to emit an initial 'valid' event if `isRerun`?